src/packages/core/src/charts/chart-common.ts

```diff
@@ -1,17 +1,59 @@
-import { Vega } from "@widget-editor/types";
+import { Vega, Generic } from "@widget-editor/types";
 
 export default class ChartCommon {
   wConfig: any;
+  wData: Generic.ObjectPayload = [];
```

Comment:
this is good, just so you know, the idea of the Generic namespace is when we are not sure what comes in, instead of using `any` this will give us better info within the compiler @clementprdhomme   if you need to use any, usually it should go into the generic one :)